src/cc65/declare.c

```diff
@@ -490,7 +490,7 @@ static void ParseEnumDecl (void)
         }
 
         /* Add an entry to the symbol table */
-        AddConstSym (Ident, type_int, SC_ENUM, EnumVal++);
+        AddConstSym (Ident, type_int, SC_ENUM|SC_CONST, EnumVal++);
```

Comment:
As far as I can see our style is to have blanks before and after binary operators like `|`. As you usually perfectly follow the given style it seems appropriate to ask for an adjustment here.

Reply:
I chose to follow the "local style". Fixed anyways. (Same below.)

Reply:
> I chose to follow the "local style". 

Which from my POV is in general the most professional approach!

> Fixed anyways.

I think that cc65 has in general such a style-consistent code base, that it makes sense here, so thanks :-)